client/go/vespa/target.go

```diff
@@ -208,11 +208,34 @@ type cloudTarget struct {
 	tlsOptions TLSOptions
 	logOptions LogOptions
 
-	queryURL       string
-	documentURL    string
+	urlsByCluster  map[string]string
 	authConfigPath string
 }
 
+func (t *cloudTarget) resolveEndpoint(cluster string) (string, error) {
+	if cluster == "" {
+		for _, u := range t.urlsByCluster {
```

Comment:
Isn't this loop unnecessary?

Reply:
How would I get the value of the entry, if the map contains exactly one entry?  I found this which seems ugly:

```
t.urlsByCluster[reflect.ValueOf(t.urlsByCluster).MapKeys()[0]]
```